extension/extension_test.go

```diff
@@ -25,6 +25,29 @@ func TestCompareVersions(t *testing.T) {
 	assert.Equal(t, -1, CompareVersions("zugzug.1.1", "1.1.daboo"))
 }
 
+// Helper function to replace the original FilterForUpdates method
+func filterForUpdates(extensions Extensions, allExtensionsMap *ExtensionsMap) Extensions {
```

Comment:
Code in `controller/controller.go` is still calling FilterForUpdates on Extension object. I'm curious why test is using this helper method?